workers/ohsome_quality_analyst/indicators/density/indicator.py

```diff
@@ -66,59 +65,96 @@ def create_figure(self) -> None:
+
+        # Create figure and add traces
+        fig = go.Figure()
+
+        fig.add_trace(
+            go.Scatter(
+                x=x,
+                y=fill_1,
+                mode="lines",
+                fill="tonexty",
+                line=dict(color="red"),
+                showlegend=False,
+            )
         )
-        plt.setp(line, linestyle="--")
-
-        line = ax.plot(
-            x,
-            y2,
-            color="black",
-            label="Threshold B",
+        fig.add_trace(
+            go.Scatter(
+                x=x,
+                y=fill_2,
+                mode="lines",
+                fill="tonexty",
+                line=dict(color="yellow"),
+                showlegend=False,
+            )
+        )
+        fig.add_trace(
+            go.Scatter(
+                x=x,
+                y=fill_3,
+                mode="lines",
+                fill="tonexty",
+                line=dict(color="green"),
+                showlegend=False,
+            )
         )
-        plt.setp(line, linestyle=":")
-
-        # Fill in space between thresholds
-        ax.fill_between(x, y2, 0, alpha=0.5, color="red")
-        ax.fill_between(x, y1, y2, alpha=0.5, color="yellow")
-        ax.fill_between(x, y1, max(max(y1), self.count), alpha=0.5, color="green")
-
-        # Plot point as circle ("o").
-        ax.plot(
-            self.area_sqkm,
-            self.count,
-            "o",
-            color="black",
-            label="location",
+        fig.add_trace(
+            go.Scatter(
+                x=x,
+                y=y2,
+                mode="lines",
+                line=dict(dash="dot", color="black"),
+                name="Threshold B",
+            )
+        )
+        fig.add_trace(
+            go.Scatter(
+                x=x,
+                y=y1,
+                mode="lines",
+                line=dict(dash="dash", color="black"),
+                name="Threshold A",
+            )
+        )
+        fig.add_trace(
+            go.Scatter(
+                x=[self.area_sqkm],
+                y=[self.count],
+                mode="markers",
+                marker=dict(symbol="circle", size=10, color="black"),
+                name="Location",
+            )
+        )
+
+        # Update layout
+        fig.update_layout(
+            title="Density (POIs per Area)",
+            xaxis_title="Area ($km^2$)",
+            yaxis_title="POIs",
+            template="plotly_dark",
         )
 
-        ax.legend()
+        raw = fig.to_dict()
+        raw["layout"].pop("template")  # remove boilerplate
+        self.result.figure = raw
 
-        img_data = StringIO()
-        plt.savefig(img_data, format="svg")
-        self.result.svg = img_data.getvalue()  # this is svg data
-        plt.close("all")
+        # Legacy support for SVGs
+        img_bytes = fig.to_image(format="svg")
+        self.result.svg = img_bytes.decode("utf-8")
```

Comment:
If easily possible, the svg shouldn't have a black background:
![test](https://user-images.githubusercontent.com/2878899/231525924-70c768b2-534e-4037-af85-7b0a487028a8.svg)

Reply:
done